src/www/ui/user-add.php

```diff
@@ -85,12 +85,28 @@ function Add() {
       $text = _("Passwords did not match. Not added.");
       return ($text);
     }
+    
+    if(empty($Email))
+    {
+      $text = _("Email must be specified. Not added.");
+      return ($text);
+    }
+
     /* Make sure email looks valid */
     $Check = preg_replace("/[^a-zA-Z0-9@_.+-]/", "", $Email);
+
     if ($Check != $Email) {
       $text = _("Invalid email address.  Not added.");
       return ($text);
     }
+    
+    /* Make sure email is unique */
+    $email_count = $this->dbManager->getSingleRow("SELECT * FROM users WHERE user_email = '".$Email."';");
+    if ($email_count > 1) {
```

Comment:
That's what I wanted to use in the first place, but it didn't work somehow. Gonna try that again.